knip.ts

```diff
@@ -0,0 +1,57 @@
+import type { KnipConfig } from "knip";
+
+const config: KnipConfig = {
+  ignore: ["**/*.test.ts", "**/*.spec.ts", "packages/nuxt/playground/**/*"],
+  workspaces: {
+    ".": {
+      entry: "checkly.config.ts",
+    },
+    "apps/web": {
+      entry: ["contentlayer.config.ts", "lib/trpc/index.ts", "trpc.config.ts"],
+    },
+    "apps/api": {
+      entry: "src/worker.ts",
+      ignore: ["**/*.test.ts", "src/pkg/testutil/*.ts"],
+    },
+    "internal/billing": {
```

Comment:
Could we do a catchall here?
"
```suggestion
    "internal": {
      "./*/src/index.ts"
    }
```

Reply:
no idea if that would work

Reply:
unfortunately not cause we have a couple of packages in there that need to be ignored (like tinybird) and I don't think we can say 'all folders except x' with glob